backend/packages/Upgrade/src/api/services/ExperimentExcludeService.ts

```diff
@@ -0,0 +1,85 @@
+import { Service } from 'typedi';
+import { OrmRepository } from 'typeorm-typedi-extensions';
+import { ExplicitExperimentGroupExclusionRepository } from '../repositories/ExplicitExperimentGroupExclusionRepository';
+import { ExplicitExperimentIndividualExclusionRepository } from '../repositories/ExplicitExperimentIndividualExclusionRepository';
+import { ExplicitExperimentIndividualExclusion } from '../models/ExplicitExperimentIndividualExclusion';
+import { ExplicitExperimentGroupExclusion } from '../models/ExplicitExperimentGroupExclusion';
+import { ExperimentService } from './ExperimentService';
+import { Experiment } from '../models/Experiment';
+
+@Service()
+export class ExperimentExcludeService {
+  constructor(
+    @OrmRepository()
+    private explicitExperimentGroupExclusionRepository: ExplicitExperimentGroupExclusionRepository,
+    @OrmRepository()
+    private explicitExperimentIndividualExclusionRepository: ExplicitExperimentIndividualExclusionRepository,
+    public experimentService: ExperimentService
+  ) {}
+
+  public getAllExperimentUser(): Promise<ExplicitExperimentIndividualExclusion[]> {
+    return this.explicitExperimentIndividualExclusionRepository.findAllUsers();
+  }
+
+  public getExperimentUserById(userId: string, experimentId: string): Promise<ExplicitExperimentIndividualExclusion> {
+    return this.explicitExperimentIndividualExclusionRepository.findOneById(userId, experimentId);
+  }
+
+  public async experimentExcludeUser(userIds: Array<string>, experimentId: string): Promise<ExplicitExperimentIndividualExclusion[]> {
+    const experiment: Experiment = await this.experimentService.findOne(experimentId);
+    if(!experiment) {
+      throw new Error('experiment not found');
+    }
+
+    let ExplicitExperimentIndividualExcludeDoc = new ExplicitExperimentIndividualExclusion();
```

Comment:
camelCase here `explicitExperimentIndividualExcludeDoc`

Reply:
@YashilDepani is this done?